README.md

```diff
@@ -1,44 +1,52 @@
-# [play floppy bird](https://nebez.github.io/floppybird/)
-
-If you missed the Flappy Bird hype, here's your chance to try the best vintage knockoff.
-
-**Features**
-
-* 🎉 good ol' div's for all the objects and graphics,
-* 🖥 scales perfectly on almost any screen, both mobile and desktop,
-* 💩 unoptimized, laggy, and not nearly as fast as a canvas implementation,
-
-Enjoy.
-
-https://nebez.github.io/floppybird/ (or play [**easy mode**](https://nebez.github.io/floppybird/?easy))
-
-### Clones
-
-* https://wanderingstan.github.io/handybird/
-    * **[@wanderinstan](https://github.com/wanderingstan)** enables hand gestures to play using doppler effect and a microphone
-* http://www.hhcc.com/404
-    * **[Hill Holiday](http://www.hhcc.com/)** using it for their 404
-* http://heart-work.se/duvchi
-    * promotional campaign for an album release
-* https://www.progressivewebflap.com/
-    * **[@jsonthor](https://twitter.com/jsonthor)** lets you take floppy bird with you as a progressive web app
-* https://github.com/rukmal/FlappyLeapBird
-    * **[Rukmal](http://rukmal.me/)** integrates the LeapMotion Controller
-* http://chrisbeaumont.github.io/floppybird/
-    * **[@chrisbeaumont](https://github.com/chrisbeaumont)** puts the bird on auto-pilot
-* http://www.lobe.io/flappy-math-saga/
-    * **[@tikwid](https://github.com/tikwid)** teaches you math
-* http://dota2.cyborgmatt.com/flappydota/
-    * flappy dota
-* http://labs.aylien.com/flappy-bird/
-    * **[@mdibaiee/flappy-es](https://github.com/mdibaiee/flappy-es)** brings skynet to floppy bird
-* https://emu.edu/gaming-hub/flappy-huxman-game/
-    * university celebrates 100 years by putting President Susan Huxman on a floppy bird body
-
-### Notice
-
-The assets powering the visual element of the game have all been extracted directly from the Flappy Bird android game. I do not own the assets, nor do I have explicit permission to use them from their creator. They are the work and copyright of original creator Dong Nguyen and .GEARS games (http://www.dotgears.com/).
-
-I took this Tweet (https://twitter.com/dongatory/status/431060041009856512 / http://i.imgur.com/AcyWyqf.png) by Dong Nguyen, the creator of the game, as an open invitation to reuse the game concept and assets in an open source project. There is no intention to steal the game, monetize it, or claim it as my own.
-
-If the copyright holder would like for the assets to be removed, please open an issue to start the conversation.
+Floppy Bird
```

Comment:
Why did this file change?

Reply:
I added the SDK to Jerobi's existing floppybird code. There have been changes to the main repo in the time since Jerobi first cloned it